src/AjaxDynamicDataTable.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DynamicDataTable from "./DynamicDataTable";
+import {AxiosInstance as axios} from "axios";
+
+class AjaxDynamicDataTable extends Component {
+
+    constructor(props) {
```

Comment:
There is no need to pass props to the constructor unless using it to set initial state

Reply:
If I were to do this, do I need to change the call to `super(props)` or simply remove it?

Reply:
You still need to call super but without the props parameter

Reply:
As follows?

```
constructor() {
    super();
    // ...
}

Reply:
Bingo 👍

Reply:
Done.